packages/devtools_shared/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # 10.0.0-dev.2
 * Support detecting package roots for nested Dart projects in the
 `packageRootFromFileUriString` utility method.
+* Added helper `deserialize`
+* Extended serialization for `HeapSample` and `ExtensionEvents`
```

Comment:
10.0.0-dev.2 is already published. Please add a new section for dev.3 and bump the pubspec version. Then add these entires to the dev.3 section